scanpipe/config.py

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
IMHO we need a better name as this is not a general purpose configuration. It could be either:
- `ecosystem_config.py`
- or better `d2d_config.py` as this is really focused on d2d for now, and may even need to be brought down one dir under `/pipelines`

Reply:
Done, thanks!